app/assets/stylesheets/layout_components/accordions.scss

```diff
@@ -52,27 +52,46 @@
 }
 
 .accordion {
+  border-top: 1px solid $smoke;
+  border-bottom: 1px solid $smoke;
+  background: none;
+
   &__header {
-    background: $brand-blue-light;
     overflow: hidden;
     display: block;
+    padding: $base-padding/1.5;
+    padding-left: $base-padding;
+    font-weight: 600;
+
+    &--arrow {
+      font-size: $global-font-size*1.5;
+      display: block;
+      margin: 0 auto;
+
+        @include breakpoint(medium) {
+        float: left;
+        margin-right: $base-margin;
+      }
+    }
+
+    &:hover {
+      background: $ash;
+      color: darken($brand-blue-light, 1%);
+    }
   }
 
   &__heading {
-    @extend .heading;
-    @include small-10;
-  }
+    text-align: center;
+    font-size: $global-font-size;
 
-  &__toggle {
-    @include small-2;
-    text-align: right;
-    color: $white;
-    padding-top: $base-padding*1.25;
+    @include breakpoint(medium) {
```

Comment:
In all the places we are using `breakpoint(medium)`, can we instead swap them and use `@include breakpoint(small down)` or `@include breakpoint(small only)`?

I think that will make things more readable and follow a pattern we have in ama_styles. (We use `breakpoint(medium)` a bit, but predominatly we are using `@include breakpoint(small...`)

Reply:
After our conversation, we decided that for a mobile-first design, using a medium breakpoint would be best here